gazelle/common/treesitter/grammars/kotlin/BUILD.bazel

```diff
@@ -0,0 +1,19 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+go_library(
+    name = "kotlin",
+    srcs = ["binding.go"],
+    cdeps = [":kotlin_cc"],
+    cgo = True,
+    copts = ["-Iexternal/tree-sitter-kotlin"],  #keep
```

Comment:
For some reason gazelle removes this line in the aspect-cli repo unless the `#keep` is added